File: backend/src/test/java/org/globe42/CrypticApplicationTest.java
package org.globe42;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.TestPropertySource;
import org.springframework.test.context.junit.jupiter.SpringExtension;

@ExtendWith(SpringExtension.class)
@SpringBootTest
@TestPropertySource("/test.properties")
public class CrypticApplicationTest {

	@Test
	public void contextLoads() {
	}

}

File: backend/src/test/kotlin/org/globe42/CrypticApplicationTest.kt
package org.globe42

import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.TestPropertySource

@SpringBootTest
@TestPropertySource("/test.properties")
class CrypticApplicationTest {

    @Test
    fun `context loads`() {
    }

}
